dspace/config/dspace.cfg

```diff
@@ -182,6 +182,9 @@ mail.allowed.referrers = ${dspace.hostname}
 mail.message.headers = subject
 mail.message.headers = charset
 
+# Helpdesk telephone.  Not email, but should be with other contact info.
+telephone.helpdesk = +1 555 555 5555
```

Comment:
Should we consider renaming this configuration something like `mail.message.telephone` since it's only used in mail templates? I'm also fine with it named as-is, but we probably should make it clear in the comment that it's only used by the mail templates, and not displayed in UIs, etc.

Reply:
I didn't like the name either, but also disliked all the alternatives I had thought of up to then.  `mail.helpdesk.telephone`?

Reply:
I would argue that the helpdesk phone number might be thought by some to be useful in UI pages and adopted for local use there, so we shouldn't be too quick to comment that it isn't.  But I've renamed it and made it optional.